schedule-tracking/src/main/java/org/motechproject/scheduletracking/domain/json/AlertRecord.java

```diff
@@ -5,15 +5,24 @@
 import java.util.ArrayList;
 import java.util.List;
 
+/**
+ * <code>AlertRecord</code> is used to deserialize alert from json format.
```

Comment:
it is not used to "deserialize"; it just keeps the deserialized form of an alert